lib/pure/os.nim

```diff
@@ -800,6 +800,8 @@ proc getHomeDir*(): string {.rtl, extern: "nos$1",
   ##
   ## This proc is wrapped by the `expandTilde proc <#expandTilde,string>`_
   ## for the convenience of processing paths coming from user configuration files.
+  ## Its available for JavaScript, NodeJS and NimScript backends but at compile-time only.
+  ## Its available for C, C++ and ObjectiveC backends.
```

Comment:
As for here, it might be better to say something like:
*Note:* For the JavaScript, node.js, and NimScript backends, this proc is only available at compile-time.

...and I would leave out anything regarding the C, C++, and Obj-C backends.